app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self,
+                 name: str,
+                 age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for person in people:
+        new_person = Person(person["name"], person["age"])
+        person_list.append(new_person)
+
+    for human in people:
+        if human.get("wife") is not None and human["wife"] is not None:
```

Comment:
```suggestion
        if human.get("wife") is not None:
```

The second part of check is almost identical to the first

Reply:
The same with husband